Src/Domain/Entities/LanguageScore/LanguageCertification.cs

```diff
@@ -5,13 +5,13 @@ namespace Domain.Entities.LanguageScore;
 public sealed class LanguageCertification:BaseEntity
 {
     public string Name { get; private set; }
-    public bool? IsActive { get; private set; }
+    public bool IsActive { get; private set; }
```

Comment:
Why didn't you give a default value to this instead of the optional parameter in the constructor?
    public bool IsActive { get; private set; } = false; 
    @MehranLabour

Reply:
initializing the IsActive property to false is redundant because in C#, the default value of a bool (boolean) property is false.

what is your opinion?
@mahdignb